src/MockBot.Api/Services/Dmr/DmrServiceSettings.cs

```diff
@@ -12,6 +12,11 @@ public class DmrServiceSettings
         private const int DefaultHttpRequestTimeoutMs = 30_000;
         private const int DefaultDmrRequestProcessIntervalMs = 5_000;
 
+        /// <summary>
+        /// The Id of this running instance of the MockBot.
+        /// </summary>
+        public string BotId { get; set; }
```

Comment:
The `DmrServiceSettings` object contains specific settings for the DMR API, I think this `BotName` property should live in a separate object. Since this is Mock Bot code, you could just put the property in an `AppSettings` object since the BotName configures this very application.

Reply:
Yep - quite agree, moved.